clang/test/SemaSYCL/delayed-parsing.cpp

```diff
@@ -1,7 +1,9 @@
-// RUN: %clang_cc1 -std=c++1y -verify -fsyntax-only %s
-// RUN: %clang_cc1 -std=c++1y -verify -fsyntax-only %s -DDELAYED_TEMPLATE_PARSING -fdelayed-template-parsing
-// RUN: %clang %s -DDELAYED_TEMPLATE_PARSING -fsycl -c
-// REQUIRES: system-windows
+// RUN: %clang_cc1 -std=c++1y -verify -fsyntax-only %s -triple x86_64-unknown-windows-msvc
+// RUN: %clang_cc1 -std=c++1y -verify -fsyntax-only %s -DDELAYED_TEMPLATE_PARSING -fdelayed-template-parsing -triple x86_64-unknown-windows-msvc
+
```

Comment:
Why this test is in `clang/test/SemaSYCL`? There no options enabling SYCL specific behavior.

I don't this this test is needed.
The only change #541 does is in the driver and it's covered by driver test and one pre-processor test `clang/test/Preprocessor/sycl-macro.cpp`. These two should be enough. 

I suggest removing the rest of the tests added in #541.